src/language-markdown/pragma.js

```diff
@@ -1,11 +1,11 @@
 import parseFrontMatter from "../utils/front-matter/parse.js";
 
-const pragmas = ["format", "prettier"];
+const FormatPragmas = ["format", "prettier"];
+const NoFormatPragmas = ["noformat", "noprettier"];
```

Comment:
```suggestion
const FORMAT_PRAGMAS = ["format", "prettier"];
const NO_FORMAT_PRAGMAS = FORMAT_PRAGMAS.map(pragma => `no-${pragma}`);
```

Maybe we should put this constants somewhere so all plugins can use it.

Reply:
Done in #17346, please rebase.

Reply:
Thanks, I rebased and tried to follow this pattern. I may not have understood the intent with the FIXME comments that were added; since I just added additional ignore code, I removed those FIXMEs.

Reply:
> the FIXME comments that were added

Sorry, mistake. Thanks for fixing them.